docs/reference/commandline/run.md

```diff
@@ -255,6 +256,15 @@ Docker daemon.
 
 For in-depth information about volumes, refer to [manage data in containers](https://docs.docker.com/engine/tutorials/dockervolumes/)
 
+### Mount using the new interface (--mount)
+
+The new mount interface is described in the [service create command section](service_create.md).
```

Comment:
I'd like to avoid using "new" in documentation, as it's no longer "new" in the next release. Something like;

```Markdown
### Add bin-mounts or volumes using the --mounts flag

The `--mounts` flag allows you to mount volumes, host-directories and `tmpfs`
mounts in a container.

The `--mount` flag supports most options that are supported by the `-v` or
`--volume` flag, but uses a different syntax. For in-depth information on the
`--mount` flag, and a comparisson between `--volume` and `--mount`, refer to
the [service create command
reference](service_create.md#add-bind-mounts-or-volumes).
```

But I'm not very creative here, perhaps @mstanleyjones has a better suggestion :smile: